src/routes/Home.jsx

```diff
@@ -0,0 +1,46 @@
+export default function Home() {
+  return (
+    <div className="flex items-center justify-center min-h-screen text-white lg:h-screen bg-gradient-to-t from-slate-400 to-slate-600">
+      <div className="w-4/6 p-5 mt-6 mb-6 text-white rounded-lg sm:w-5/6 bg-slate-600 ">
+        <h2 className="mb-4 text-xl text-center md:text-5xl md:font-bold">
+          Welcome to React and Charts
+        </h2>
+        <p className="mb-6 text-center md:text-lg">
+          This hw3 uses react router to fetch data from a custom API and
+          displays the data using charts -
+          <strong> Use the navigational bar to navigate to a page</strong>
+        </p>
+        <h3 className="mb-2 font-semibold md:text-2xl">Home</h3>
+        <p className="mb-6 text-center sm:text-left md:text-lg">
```

Comment:
I notice you're repeating these classNames a lot in the same way. You can edit an index.css or whatever the main file you import into app.jsx would be to something like this...

```
@tailwind base;
@tailwind components;
@tailwind utilities;

/* 
You will need to style each base element as you use it, but this is good!
It prevents/protects us from using superflous HTML elements, but it also forces
us to use inline tailwind with classNames={`stuff`}
*/
@layer base {
  h1 {
    @apply text-4xl;
    font-weight: 700;
  }
  h2 {
    @apply text-2xl;
    font-style: oblique;
    font-size: 36px;
  }
  h3 {
    font-size: 1.75rem;
    font-weight: 500;
    margin-bottom: 8px;
  }
}
```

This allows you to pre-define a style for html elements without having to worry about getting it right the first time. They can be overridden too with `!mb-3` instead of just an `mb-3` in the className if you need to which will mark the inline css as important.

Reply:
Thanks, this was helpful and insightful. I would do this for all elements but not all of them have the same class names, one ```<p></p>``` might differ from another; however, on the elements that do, I refactored the same class names into one with pre-defining a style for that specific element.

Reply:
Also, very easy to refactor! You can hover over the tailwind class and it displays the CSS so I just copied and pasted.